### ISSUE TITLE
Test by running SimpleCnetClient  failed.

### ISSUE THREAD

**Issue by Thien223** [Vagabond-K/VagabondK.Protocols#1]
Thanks for your efforts. Sir.

I'm on a project of connecting to LS PLC using RS-485 cable. 

so, I have downloaded your projects, and run `SimpleCnetClient` (this is the only one Serial connection I have found here, other project all using TCP connection) but it did nothing.

After debugging, I've found out that the Read() function was failed because the wrong Header was received. 

![image](https://user-images.githubusercontent.com/31118781/206675036-0341d908-3984-4b49-819a-ac28521bdd5e.png)

 could you please tell me how to fix this?

Thank you so much, sir.

**Comment by Vagabond-K** [comment]
First of all, sorry for the late reply. Sir. And I'm so sorry that I can't speak English well. 

ResponseHeaderError is raised when the beginning of the response is neither ACK (0x06) nor NAK (0x15).
I think you are connecting to Modbus channel of LS PLC or connecting to LS PLC that provides protocol other than LS Cnet.

What model of LS PLC are you using?
If I know the model, I might be able to give you a more detailed answer.

Thank you so much, sir.

**Comment by Thien223** [comment]
Thank for your reply.

I found out that the SimpleCnetClient is only works with XGT Server mode. I mean the Operation Mode should be XGT server like this: 

![image](https://user-images.githubusercontent.com/31118781/208364069-00ae7a1a-c3b1-42a1-9df0-6cd703f211b6.png)

So I have created other frame format to work with Mobus RTU Server mode. It's okay now.
![image](https://user-images.githubusercontent.com/31118781/208367067-cd327fb4-5d14-46f6-a9f9-b3790d02a09c.png)

I'm using XGT PLC with XGL-C42B module, Rs-485 Serial Mobus RTU communication. 

And I used your SerialPort Class, and learn the way of constructing bytes[] request from you as well. 
Thank you so much.
